Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -540,12 +539,77 @@ theorem cliqueSet_map_of_equiv (G : SimpleGraph α) (e : α ≃ β) (n : ℕ) :
 
 end CliqueSet
 
+/-! ### Clique number -/
+
+
+section CliqueNumber
+
+variable {α : Type*} {G : SimpleGraph α}
+
+/-- The maximal number of vertices in a graph `G`. -/
+noncomputable def cliqueNum (G : SimpleGraph α) : ℕ := sSup {n | ∃ s, G.IsNClique n s}
+
+/-- A maximum clique in a graph `G` is a clique with the largest possible size. -/
+structure IsMaximumClique (G : SimpleGraph α) (s : Finset α) : Prop where
+  (clique : G.IsClique s)
```

Comment:
```suggestion
  (isClique : G.IsClique s)
```

Reply:
Should it then not also be named `isClique` in `structure IsNClique`?

Reply:
Yes it should!

Reply:
Hope it's fine if I add that change to this PR